app/Models/Ical.php

```diff
@@ -238,6 +238,12 @@ public function getPeriodInfo(\DatePeriod $datePeriod)
                 continue;
             }
 
+            // Undefined index
+            // occurs when the last event of the previous month continues past midngiht,
+            // into the current month
```

Comment:
Comments should be full sentences (punctuation, capital letters, ...)

s/midngiht/midnight/

Reply:
Done